_installation/interoperability.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Single vs. multiple database setups
+nav_order: 5
+---
+## Using Both TDS/T-SQL and PostgreSQL with Babelfish
+
+Babelfish provides T-SQL semantics on top of PostgreSQL using the T-SQL dialect via the TDS port. You can also access the Babelfish database through the standard PostgreSQL port with PostgreSQL SQL statements. If you use both (TDS and PostgreSQL) ports and dialects, interference between PostgreSQL and Babelfish may occur, potentially affecting syntax, semantics, and compatibility with future releases of Babelfish. As an administrator or user you should be aware of potential issues related to schema names, identifiers, permissions, transactional semantics, multiple result sets, default collations, and future Babelfish upgrades.
```

Comment:
Please keep the original phrase "PostgreSQL statements or PostgreSQL access occur in the context of Babelfish": the current wording over-simplifies the issue. This nuance matters